docs/txp.md

```diff
@@ -45,17 +45,19 @@ associated with the hash of the header of the last applied block
 (i. e. this hash is integral part of this state). So, given a sequence
 of blocks `B₀, B₁, B₂, …` we can compute sequence of states `S₀, S₁,
 S₂, …`. There is also genesis state `G` which takes places before any
-block is applied. This state is derived from genesis data (see ??? TODO).
+block is applied. This state is derived from genesis data (see [mainnet genesis JSON](https://raw.githubusercontent.com/input-output-hk/cardano-sl/e7cfb1724024e0db2f25ddd2eb8f8f17c0bc497f/node/mainnet-genesis.json) for reference).
```

Comment:
This file (mainnet-genesis.json) as-is won't be understood by an external reader. My idea was to put a link to the document created in CSL-1688 here, but since CSL-1688 is WIP, I left a TODO.